resources/views/dashboard.blade.php

```diff
@@ -2,8 +2,30 @@
 @section('content')
     <div class="main-panel">
         <div class="container-fluid">
-          <h4>{{ Auth::user()->firstname}}'s expenses</h4>
-            <button class="btn btn-danger openAddExpenseModal" data-toggle="modal" data-target="#addExpenseModal">Add Expense</button>
+            <div class="row">
+                <div class="col-md-2">
+                    <h4>{{ Auth::user()->firstname}}'s expenses</h4>
+                </div>
+                <div class="col-md-10">
+                    <button class="btn btn-danger openAddExpenseModal" data-toggle="modal" data-target="#addExpenseModal">Add Expense</button>
+                    <div class="btn-group">
+                        <button class="btn" type="button" data-toggle="dropdown"><i class="material-icons">person_add</i></button>
+                        <ul class="dropdown-menu" >
+                            @foreach($friendNotifications as $fnotification)
```

Comment:
This is the one im talking about. This would break other views because they are not required to pass in a notifications variable